src/sentry/incidents/subscription_processor.py

```diff
@@ -708,43 +718,19 @@ def get_anomaly_data_from_seer(self, aggregation_value: float | None):
             )
         except (TimeoutError, MaxRetryError):
             logger.warning(
-                "Timeout error when hitting anomaly detection endpoint",
-                extra={
-                    "subscription_id": self.subscription.id,
-                    "dataset": self.subscription.snuba_query.dataset,
-                    "organization_id": self.subscription.project.organization.id,
-                    "project_id": self.subscription.project_id,
-                    "alert_rule_id": self.alert_rule.id,
-                },
+                "Timeout error when hitting anomaly detection endpoint", extra=extra_data
             )
             return None
 
-        if response.status != 200:
+        if response.status > 400:
             logger.error(
-                f"Received {response.status} when calling Seer endpoint {SEER_ANOMALY_DETECTION_ENDPOINT_URL}.",  # noqa
-                extra={"response_data": response.data},
+                "Error when hitting Seer detect anomalies endpoint",
```

Comment:
Why don't we want to log the response error message here?

Reply:
Is it that each status code has a specific meaning in Seer?

Reply:
Oh I can add that back. I was thinking that since the Seer response should have the `message` in it for an error we'd use that.